src/quo/components/tags/collectible_tag/schema.cljs

```diff
@@ -0,0 +1,15 @@
+(ns quo.components.tags.collectible-tag.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
```

Comment:
We prefer open maps, because sometimes extra data might be passed down to components.

Malli maps are open by default, so removing `{:closed true}` would do.

Reply:
I don't fully agree with this, even though this was already merged. I think we should prefer closed maps for component props, unless needed for the "extra data" edge-case (which IMO shouldn't even happen 🥲). We had the conversation on discord a while back, but we didn't finish it and I think we should continue it someplace else, maybe an issue/discussion on GH, since on discord it's difficult to keep track of simultaneous convos.

Reply:
Yeah let's centralise it somewhere. Discord vote, notion doc or a GitHub discussion. Or anything like this 👌

Reply:
here it is https://github.com/status-im/status-mobile/discussions/18842